website/views/core.py

```
@@ -572,240 +574,257 @@ def search(request, template="search.html"):
+                                    "repos": "repos",
+                                    "languages": "repos",  # languages search returns repos
+                                }
+                                key = type_to_key.get(search_type, search_type)
+                                items = context.get(key)
+                                if items:
+                                    if hasattr(items, "count"):
+                                        result_count = items.count()
+                                    elif isinstance(items, list):
+                                        result_count = len(items)
+
+                        except Exception as e:
```

Ruff (0.14.7)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)